src/Estimators/tests/test_manager_mpi.cpp

```diff
@@ -23,7 +23,7 @@ namespace testing
 {
 bool EstimatorManagerNewTest::testMakeBlockAverages()
 {
-  if (em.my_comm_->rank() == 1)
+  if (em.my_comm_->rank() == 0)
```

Comment:
Could you explain this change? The change doesn't seem to be complex related but more like a logic error?

Reply:
The test should work for any number of mpi ranks but due to this typo it works for any legal number of ranks except 1.

Reply:
Aha. Got it.

Reply:
Also in this case in the commit comment by complex I meant complicated estimators.

Reply:
I see. Maybe adjust title?